device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/IotHubStatusCode.java

```diff
@@ -97,10 +162,10 @@ public static IotHubStatusCode getIotHubStatusCode(int httpsStatus)
                 iotHubStatus = UNAUTHORIZED;
                 break;
             case 403:
-                iotHubStatus = TOO_MANY_DEVICES;
+                iotHubStatus = QUOTA_EXCEEDED;
                 break;
```

Comment:
Can this be combined with the case on 108? If you can't double them up like one can in .NET, I'd at least put them next to each other.

Reply:
Same with case 412

Reply:
Different cases that have the same action taken.

Reply:
Not unless we are consolidating these enum values then, no. A user's application may treat a SERVER_BUSY and a THROTTLED case the same way, but there's no reason for this particular class to do that, too. The ```isRetryable``` flag on the exception that holds this enum will do the heavy lifting on that.

Reply:
Never mind, bad read.